compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -14023,6 +14026,13 @@ TR::Register *OMR::Z::TreeEvaluator::inlineVectorBinaryOp(TR::Node *node, TR::Co
     TR_ASSERT_FATAL_WITH_NODE(node, node->getDataType().getVectorLength() == TR::VectorLength128,
         "Only 128-bit vectors are supported %s", node->getDataType().toString());
 
+    // Before z14, vector instructions for floating point operations supported only double-precision values.
```

Comment:
I think changes are OK - as long as @ehsankianifar  has addressed all the recommendations from Gita. 

Just a nitpick / recommendation, on IBM Z, it is short-format for single precision binary floating point values and long format for double precision binary floating point values. That is what we use here in the comment instead of single precision / double precision.